README.md

```diff
@@ -1,2 +1,44 @@
-# project-team-2-news-website
-Project Team 2 News Website
+# NEWS APP [TEAM_2]
+- Simple news app 
+
+## ✧ Links
+- App Live link: [Here]( https://gsg-g11.github.io/news-team-2/)
+- App Repo Link on Github:[Here](https://github.com/GSG-G11/news-team-2)
+- Figma Design link: [Here](https://www.figma.com/file/LsEI9IHnLKywTLHhXrDlfd/News-Website-UI-and-Presentation-for-Opportunists-(Community)?node-id=0%3A1) 
+
+## ✧ Product Main Features:
+* User-friendly.
+* Desktop First.
+* Responsive design.
+
+## ✧User Stories
+After different sessions with the mentors, the mentors developed the following user stories: <br>
+1- As a `Person`, I can ***read the latest news*** so that ***I quickly know what's going on around the world.***. <br>
+2- As a `Person`, I can *** see the different news sections and each section should contain its related news (e.g. Politics, Sport, Economy, Tech ...etc )***  <br>
+3- As a `Person`, I can *** looking to read the top blogs***  <br>
+4- As a `Person`, I can *** reading the news in detail***  <br>
+5- As a `Person`, I can *** easily view the site when browsing on my mobile, tablet or desktop & understand the sites content easily on whatever device I choose to view it in. ***  <br>
+
+## ✧How to install the project to run it locally.
+
+on terminal/bash
+
+```shell
+git clone https://github.com/GSG-G11/news-team-2
+cd news-team-2
+go live server
+```
+
+## ✧Technology
+- HTML ➙ to build the structure of the pages 
+- CSS ➙ to style the app
+
```

Comment:
add JS to toggle bar button on mobile screen

Reply:
ok, I will add it :)